en/lessons/extracting-illustrated-pages.md

```diff
@@ -0,0 +1,479 @@
+
+As you update a collection, HT keeps track of the associated metadata for each item in it. I have included in the lesson files the metadata for a sample lesson in JSON format. If you wanted to use the file from your own HT collection, you would navigate to your collections page and hover on the metadata link on the left to bring up the option to download as JSON as seen in the following screenshot.
+
+{% include figure.html filename="download-ht-json.png" caption="Screenshot of how to download collection metadata in JSON format." %}
+
+When you have downloaded the JSON file, simply move it to the directory where you placed the Jupyter notebooks. Replace the name of the JSON file in the HT notebook with the name of your collection's file.
+
+The notebook shows how to use a list comprehension to get all the `htitem_id` strings within the `gathers` object that contains all the collection information.
+
+```Python
+# you can specify your collection metadata file here
+metadata_path = "554050894-1535834127.json"
+
+with open(metadata_path, "r") as fp:
+    data = json.load(fp)
+
+# a list of all unique ids in the collection
+vol_ids = [item['htitem_id'] for item in data['gathers']]
+```
+
+<div class="alert alert-warning">
+  Tutorials often show you how to process one example item (often of a trivial size or complexity). This is pedagogically convenient, but it means you are less equipped to apply that code to multiple items&mdash;by far the more common use case. In the notebooks you will see how to encapsulate transformations applied to one item into <i>functions</i> that can be called within a loop over a collection of items.
+</div>
+
+## Visual Feature: IMAGE_ON_PAGE
+
+Given a list of volumes, we want to explore what visual features they have at the page level. The [most recent documentation](https://www.hathitrust.org/documents/hathitrust-data-api-v2_20150526.pdf) (2015) for the Data API describes a metadata object called `htd:pfeat` on pages 9-10. `htd:pfeat` is shorthand for "HathiTrust Data API: Page Features."
+
+> * `htd:pfeat`­ - the page feature key (if available):
+>    - CHAPTER_START
+>    - COPYRIGHT
+>    - FIRST_CONTENT_CHAPTER_START
+>    - FRONT_COVER
+>    - INDEX
+>    - REFERENCES
+>    - TABLE_OF_CONTENTS
+>    - TITLE
+
+What the `hathitrust-api` wrapper does is make the full metadata for a HT volume available as a Python object. Given a volume's identifier, we can request its metadata and then drill down through the page *sequence* into page-level information. The `htd:pfeat` *list* is associated with each page in a volume and in theory contains all features that apply to that page. In practice, there a quite a few more feature tags than the eight listed above. The one we will be working with is called IMAGE_ON_PAGE and is more abstractly visual than structural tags such as CHAPTER_START.
```

Comment:
Generally you'll want to enclose the names of variables, keys, column names, functions etc. in backticks, so that you get `IMAGE_ON_PAGE` instead of IMAGE_ON_PAGE. Take a quick look through and add those as necessary.

Reply:
brilliant! thanks! there were just a few of these running around